src/vhost_user/master.rs

```diff
@@ -125,6 +126,12 @@ impl Master {
 
         Ok(Self::new(endpoint, max_queue_num))
     }
+
+    /// Set the header flags that should be applied to all following messages.
+    pub fn set_flags(&self, flags: VhostUserHeaderFlag) {
```

Comment:
@jiangliu maybe I should call it `set_hdr_flags()`, let me know what you think.

Reply:
yes, set_hdr_flags() is more clear.

Reply:
Done!